packages/coverage-v8/src/provider.ts

```diff
@@ -497,27 +486,41 @@ export class V8CoverageProvider extends BaseCoverageProvider implements Coverage
 
   private async convertCoverage(
     coverage: RawCoverage,
-    projectName?: ProjectName,
-    transformMode?: 'web' | 'ssr',
+    project: WorkspaceProject = this.ctx.getCoreWorkspaceProject(),
+    transformMode?: keyof CoverageFilesByTransformMode,
   ): Promise<CoverageMap> {
-    const viteNode
-      = this.ctx.projects.find(project => project.getName() === projectName)
-        ?.vitenode || this.ctx.vitenode
     const fetchCache = transformMode
-      ? viteNode.fetchCaches[transformMode]
-      : viteNode.fetchCache
+      ? project.vitenode.fetchCaches[transformMode === 'browser' ? 'web' : transformMode]
```

Comment:
`fetchCache` is not populated for the browser, this assumes that jsdom test and browser tests have the same transforms, but browser tests don't apply the SSR transform (`__vite_ssr_1__` and such)

Reply:
Yup, for browser mode there won't be matches for `fetchCache`. Maybe it's safer to add empty/dummy fetchCache here so that they don't mix when browser mode is used via `poolMatchGlobs`.

Reply:
Yeah, I don't think using web is safe enough